src/java/org/apache/cassandra/config/Config.java

```diff
@@ -853,6 +853,8 @@ public static void setClientMode(boolean clientMode)
     public volatile DurationSpec repair_state_expires = DurationSpec.inDays(3);
     public volatile int repair_state_size = 100_000;
 
+    public boolean jmx_hide_non_java_exceptions = true;
```

Comment:
Oh also in my branch I removed the validation of the config file as it is not needed anymore, the two problematic properties were set with `0ms` default value. Not being null anymore doesn't allow to have the problem that the validation was trying to prevent. Let me know if it can be part of this patch or you want me to pull it. Just int doesn't make sense to read one more time the config file when not actually needed anymore.

Reply:
Oh I have bad sad news. I pushed my commit and then got an email from Circle that Java11 build failed:
```
_build_java:
     [echo] Compiling for Java 11...
    [javac] Compiling 2137 source files to /home/cassandra/cassandra/build/classes/main
    [javac] Note: Processing compiler hints annotations
    [javac] /home/cassandra/cassandra/src/java/org/apache/cassandra/utils/JMXServerUtils.java:67: error: package com.sun.jmx.mbeanserver is not visible
    [javac] import com.sun.jmx.mbeanserver.JmxMBeanServer;
    [javac]                   ^
    [javac]   (package com.sun.jmx.mbeanserver is declared in module java.management, which does not export it)
    [javac] Note: Processing compiler hints annotations
    [javac] Note: Writing compiler command file at META-INF/hotspot_compiler
    [javac] Note: Done processing compiler hints annotations
    [javac] 1 error
```

I feel the community won't be happy to open more internals for this :(

Reply:
hmmm, not sure how to *fix*, in the case where we let the JVM setup the server, we need to be able to inject our logic... ill try to look into another way

Reply:
have a solution by plugging deeper into jmx startup